### ISSUE TITLE
Make secure param be optionally added based on boolean value not property existence

### ISSUE THREAD

**Issue by todesignandconquer** [kahnjw/cookie-monster#17]
It would be more beneficial to add the `secure` param based on a boolean value rather than the mere existence of the property.

E.g.
```
if (opts.hasOwnProperty('secure') && opts.secure === true) newCookie += '; secure';
```

**Comment by kahnjw** [comment]
Agreed. I think even doing:

```javascript
if (opts.secure) newCookie += '; secure';
```

would work. Pull welcome.

**Comment by todesignandconquer** [comment]
https://github.com/kahnjw/cookie-monster/pull/18

**Comment by kahnvex** [comment]
Published as `0.2.1`
